lib/smart_answer_flows/marriage-abroad/outcomes/outcome_ceremonies_in_dominican_republic.govspeak.erb

```diff
@@ -57,7 +57,7 @@
                locals: { calculator: calculator } %>
     <%= render partial: 'link_to_consular_fees.govspeak.erb',
                locals: { calculator: calculator } %>
-
-    <%= render partial: 'pay_by_cash_or_credit_card_no_cheque.govspeak.erb' %>
+    <%= render partial: 'payment_for_services.govspeak.erb',
+               locals: { calculator: calculator } %>
   <% end %>
 <% end %>
```

Comment:
Based on the commit note in which it was introduced, I would've expected to see changes to the `marriage-abroad-countries-leading-to-outcome.rb` script in this commit (and others like it). Am I missing something?